src/components/Table/types.ts

```diff
@@ -22,6 +22,8 @@ export type GridTableScrollOptions =
        * How to position the row in the viewport
        */
       align?: "start" | "center" | "end";
+      /** Determines if the row should be re-scrolled to even if it's already in view (scrolled to top) */
+      forceRescroll?: boolean;
```

Comment:
@blambillotte wdyt about just having this always be true, and not exposed as an option?

I.e. cue my usual soapbox about beam having as-few-props-as-possible and ideally we just provide the "best" / same behavior all for all pages.

Reply:
yeah my initial thought was to just swap the default, but didn't get any engagement on that question in [this thread](https://homeboundinc.slack.com/archives/C01JAFC0RAN/p1734732420998599) ha
Happy to make the "don't jump"/new behavior the default and just not have a prop for it if everyone is good with that

Reply:
Yep, I think between "same behavior everywhere" bias + "no one voted / seemed to care in that thread", let's move ahead w/no customization.

Reply:
Cool okay - updated!